### PR TITLE
CRUD APIs

### PR DESCRIPTION
**Pull request by dmoygit**
PR Highlights:
1) added post, put, delete to round out the CRUD API's (please see successful postman requests below).  FYI:  please see commit 01f89df.  I used insertMany instead of save (one at a time).  It will match this first postman image. I also added a single get request: b1b9587
Screen Shot 2020-02-12 at 4 56 47 PM
Screen Shot 2020-02-12 at 4 58 11 PM
Screen Shot 2020-02-12 at 4 59 22 PM
Screen Shot 2020-02-12 at 4 59 50 PM

2) updated the README to add the API endpoints.

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -25,6 +24,52 @@ app.get('/carousel/:id/', (req, res, next) => {
   })
 });
 
+//write a post route
+app.post('/carousel', (req, res) => {
```
Comment (yerx) [inline server/index.js]: does the POST request post to a specific item page "/:refId"?

Comment (dmoygit) [reply server/index.js]: no, that's the reference id that MongoDP created (Mongo labels it '_id').  I called it refId, as in reference id, so it doesn't get confused with 'id' that Phillip used to identify the page.

**Comment by dmoygit** [comment]
I don't need an ID for a post request because it's just writing to the DB.  it doesn't need to reference a specific image.

File: server/index.js (hunk 1 above)
Comment (yerx) [reply server/index.js]: oh OK, that's why I was confused. Got it.

**Pull request merged**